modules/book.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-unused-vars */
+/* eslint-disable indent */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable object-curly-spacing */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-multiple-empty-lines */
+
+import { titleInput, authorInput, btnInput, addedBooks } from './variables.js';
+
+export class Book {
+    constructor(title, author) {
+        this.title = title;
+        this.author = author;
+    }
+}
+
+export class Store {
+    static getBooks() {
+         let books;
+         if (localStorage.getItem('books') === null) {
+             books = [];
+         } else {
+             books = JSON.parse(localStorage.getItem('books'));
+         }
+
+         return books;
+     }
+
+     static addBook(book) {
+         const books = Store.getBooks();
+         books.push(book);
+         localStorage.setItem('books', JSON.stringify(books));
+     }
+
+    static removeBook(author) {
+         const books = Store.getBooks();
+ 
+         books.forEach((book, index) => {
+             if (book.author === author) {
+                 books.splice(index, 1);
+             }
+         });
+ 
+         localStorage.setItem('books', JSON.stringify(books));
+     }
```

Comment:
- [ ] Your books are only removed from the page but not from the local storage. So when I refresh the page the removed book reappears. Kindly look into that and resolve that.